lstchain/scripts/lstchain_mc_dl1_to_dl2.py

```diff
@@ -62,6 +63,15 @@ def main():
     config = replace_config(standard_config, custom_config)
 
     data = pd.read_hdf(args.datafile, key=dl1_params_lstcam_key)
+
+    # Dealing with pointing missing values. This happened when `ucts_time` was invalid.
+    if 'alt_tel' in data.columns and 'az_tel' in data.columns:
+        # make sure there is a least one good pointing value to interp from.
+        if np.isfinite(data.alt_tel).any() and np.isfinite(data.az_tel).any():
+            impute_pointing(data)
+        else:
+            data.alt_tel = - np.pi/2.
+            data.az_tel = - np.pi/2.
```

Comment:
I'm running some more tests with this definition and it seems they work fine now.